core-feature-pack/src/main/resources/configuration/standalone/subsystems.xml

```diff
@@ -5,5 +5,6 @@
       <subsystem>logging.xml</subsystem>
       <subsystem>jmx.xml</subsystem>
       <subsystem>request-controller.xml</subsystem>
+      <subsystem>security-manager.xml</subsystem>
```

Comment:
Actually this is not accurate.  The security manager itself will only be enabled if the -secmgr flag is given at boot.